PowerPointLabs/Test/FunctionalTest/BaseFunctionalTest.cs

```diff
@@ -81,8 +81,8 @@ protected static void CheckIfClipboardIsRestored(Action action, int actualSlideN
             ShapeRange newShape = actualSlide.Shapes.Paste();
 
             // Check if pasted shape is the same as the shape added to clipboard originally
-            Assert.AreEqual(shapeNameToBeCopied, newShape.Name);
-            Assert.AreEqual(shapeToBeCopied.Count, newShape.Count);
+            //Assert.AreEqual(shapeNameToBeCopied, newShape.Name);
```

Comment:
@ChesterSng Does the tests pass if you uncomment these lines?

Reply:
This seems to be affecting another test.

Reply:
@leeyh20 oops I forgot to uncomment them. Uncommented and ShapesLabTest and SyncLabTest no problem!